pkg/model/common.proto

```diff
@@ -79,6 +79,8 @@ message ArtifactVersion {
     enum Kind {
         UNKNOWN = 0;
         CONTAINER_IMAGE = 1;
+        S3_OBJECT = 2;
+        SOURCE_CODE = 3;
```

Comment:
I feel that both of them are source code. S3 and Git are just where the source code is being stored.
@khanhtc1202 What do you think?

Reply:
I get your point. In some cases, the `object` which be stored in S3 bucket is not the source code but the binary executable file, but it's not that different as you mentioned. For now, I think we are still not clear on defining what will be the artifact for piped, so making this kind separated or as one are both understandable to me. Lets compare which one is better for users from the view side.

- Make it one: will make the kind simpler, users do not actually care about where it's stored since they can jump to the source via URL
- Separated them: will make clear (easy to get what it's from the view)

Which one do you guys think it's more acceptable?

Reply:
I think we should separate them because pipecd tracks the version of a s3 object on lambda deployment.

Reply:
> I think we should separate them because pipecd tracks the version of a s3 object on lambda deployment.

Nice point.

Reply:
So how about renaming them to

`GIT_SOURCE` and `S3_OBJECT`

to be more understandable?

Reply:
That sounds nice! I vote it!

Reply:
Nice 👍

Reply:
Ok. So lets go with it.

Reply:
I fixed, PTAL🙏